api/src/main/java/jakarta/resource/spi/security/PasswordCredential.java

```diff
@@ -104,7 +104,7 @@ void setManagedConnectionFactory(ManagedConnectionFactory mcf) {
    *  they are equal in username and password.
    *
    *  @param other  Object to which PasswordCredential is to be compared
-   *  @return <tt>true</tt> if and if the specified object is a
+   *  @return {@code true} if and if the specified object is a
```

Comment:
"if and if" probably should be "if and only if"

Reply:
how is `if...` to broken javadoc build?

Reply:
I'm not going to fix outstanding javadoc issues, that is not the goal of this PR